src/screens/level.py

```diff
@@ -137,28 +163,44 @@ def setup_objects(self, pos, obj):
         else:
             Sprite(pos, image, (self.all_sprites, self.collision_sprites))
 
-    def setup_collisions(self, pos, obj):
+        self.pf_matrix_setup_collision((obj.x, obj.y), (obj.width, obj.height))
+
+    def setup_collision(self, pos, obj):
         size = (obj.width * SCALE_FACTOR, obj.height * SCALE_FACTOR)
         image = pygame.Surface(size)
         Sprite(pos, image, self.collision_sprites)
 
-    def setup_interactions(self, pos, obj):
+        self.pf_matrix_setup_collision((obj.x, obj.y), (obj.width, obj.height))
+
+    def setup_interaction(self, pos, obj):
```

Comment:
Why remove the plurals in function names?

Reply:
That is a valid point. Though it may also confuse people into thinking they only set up _one_ of each (e.g. set up only one entity instead of setting up all needed entities)

Reply:
Oh, that's what I meant 😅
Because the functions setup_layer_tiles and setup_layer_objects iterate over all tiles / objects in a layer and call the given function for each individual element found